cfme/v2v/migration_plans.py

```diff
@@ -299,7 +300,12 @@ def is_displayed(self):
 
         def fill(self, values):
             was_change = True
-            self.run_migration.select(values)
+            warm_migration = values.get('warm_migration', False)
+            value = values.get('start_selection')
+            if warm_migration:
+                self.select_migration.select(value)
```

Comment:
so it will be more readable : 
 
                 if warm_migration:
                      self.warm_migration_button.select(value)
                else:
                      self.cold_migration_button.select(value)

Reply:
I see, I got your point, However changing self.run_migration name will impact on other TC as well.
>> test_v2v_migrations_ui.py:252:    view.schedule.run_migration.select("Save migration plan to run later")

If required, we can do a refactoring in a new PR, I do not want this PR to update any existing TC.

Reply:
Ok we can do that in later PR .